src/main/java/frc/robot/subsystems/RollerSubsystem.java

```diff
@@ -26,16 +30,34 @@ public RollerSubsystem() {
         rightBeak = MotorUtil.createTalonSRX(RIGHT_ID);
         rightBeak.follow(leftBeak);
         rightBeak.setNeutralMode(NeutralMode.Brake);
+
+        openMotor = MotorUtil.createTalonSRX(OPEN_ID);
+        openMotor.setNeutralMode(NeutralMode.Brake);
+    }
+
+    public void OpenMotor(){
+        timer.start();
     }
 
     @Override
     public void periodic() {
+        if (timer.hasElapsed(3)){
+            timer.reset();
+        }
+        else if (timer.get() < 1){
```

Comment:
can we extract these into constants at the top of this file? so it's easier to tune